crates/network-scanner/Cargo.toml

```diff
@@ -8,9 +8,12 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.79"
+get_if_addrs = "0.5.3"
 network-scanner-net ={ path = "../network-scanner-net" }
 network-scanner-proto ={ path = "../network-scanner-proto" }
+socket2 = "0.5.5"
 tokio = { version = "1.35.1", features = ["io-util"] }
+tokio-stream = "0.1.14"
```

Comment:
question: I don’t see any utility provided by this crate used in the new code. Am I missing something?

suggestion: Use `futures-core` instead.